### PR TITLE
Add nitrokey-app

### PR DESCRIPTION
**Pull request by sztsian**
Add support for nitrokey-app

Signed-off-by: zsun sztsian@gmail.com

### CODE REVIEW

**Comment by sztsian** [comment]
@1dot75cm  @cheese  Please review my SPEC first. I tested it locally but I am not sure if it is fully follows the Fedora style.
This should solved https://github.com/FZUG/repo/issues/181
And caution that, I do not have any nitrokey so I need someone else to test it.

**Comment by sztsian** [comment]
I tested again on my dayjob machine, it works. So I think there is no dependency issue in SPEC now.

**Comment by 1dot75cm** [comment]
Hi, @sztsian 
I just review `nitrokey-app.spec`. It's has some problem. I has been push it in [Gist](https://gist.github.com/1dot75cm/688e7ff7cb4ba1266718b693d606dcc4).

**Comment by sztsian** [comment]
@1dot75cm  mostly followed your advise, except the desktop file section. I modified the desktop file section based on firefox spec in Fedora.

**Comment by sztsian** [comment]
ok to test

File: rpms/nitrokey-app/nitrokey-app.spec [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Version:	0.5.1
+Release:	1%{?dist}
+Summary:	Nitrokey Configuration tool
+
+Group:		Applications/System
+License:	GPLv3
+URL:		https://github.com/Nitrokey/nitrokey-app
+#Source0:	%{url}/%{_commit}/%{repo}-%{_shortcommit}.tar.gz
+Source0:	%{url}/archive/v%{version}.tar.gz
+Source1:	https://www.nitrokey.com/sites/default/files/40-nitrokey.rules
```
Comment (1dot75cm) [inline rpms/nitrokey-app/nitrokey-app.spec]: https://github.com/Nitrokey/nitrokey-app/blob/master/data/40-nitrokey.rules
The file is already here.

File: rpms/nitrokey-app/nitrokey-app.spec [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+%description
+Nitrokey app, the application to configure Nitrokey Pro and Nitrokey Storage.
+
+%prep
+%setup -q -n %{name}-%{version}
+cp -p %SOURCE1 .
+
+%build
+%{_qt5_qmake} %{name}-qt5.pro
```
Comment (1dot75cm) [inline rpms/nitrokey-app/nitrokey-app.spec]: The `%{qmake_qt5}` is better than `%{_qmake_qt5}`.

```
$ rpm -E %qmake_qt5
```

**Review by 1dot75cm — COMMENTED** [review]
Review nitrokey-app.spec

**Comment by 1dot75cm** [comment]
ok to test

File: rpms/nitrokey-app/nitrokey-app.spec (hunk 1 above)
Comment (sztsian) [reply rpms/nitrokey-app/nitrokey-app.spec]: Yes I see, just forgot to remove Source1

**Comment by sztsian** [comment]
ok to test

**Pull request merged**